Source/WebKit/WebKitSwift/RealityKit/WKRKEntity.swift

```diff
@@ -303,8 +303,18 @@ extension WKRKEntity {
         default:
             break
         }
-        
-        guard let context = CGContext.init(data: nil, width: targetWidth, height: targetHeight, bitsPerComponent: image.bitsPerComponent, bytesPerRow: 0, space: image.colorSpace ?? CGColorSpace(name: CGColorSpace.sRGB)!, bitmapInfo: imageBitmapInfoRawValue) else {
+
+        guard
```

Comment:
```suggestion
        let context = CGContext(
            blah
             blah
        )
        guard let context else {
            stuff
        }
```

For complex expressions, I think it’s more readable for it to be split out to before the guard statement. Also, the `.init` is not needed